### ISSUE TITLE
Question about the learning rate adjustment strategy

### ISSUE THREAD

**Issue by delaprada** [microsoft/MeshTransformer#86]
Hi, thank you for providing this good work!

I have a question about the `adjust_learning_rate` method in `run_metro_handmesh.py`. I use metro to train from scratch with Freihand dataset. I found that the learning rate doesn't change and the value is always 0.0001. I refer to the [freihand_reproduce_log.txt](https://datarelease.blob.core.windows.net/metro/models/freihand_reproduce_log.txt) and also find that the learning rate is 0.0001 for the first 100 epochs and 0.00001 for the latter 100 epochs. It seems the `adjust_learning_rate` method in `run_metro_handmesh.py` doesn't work?

Thank you for your time and help!
